src/templates/addons/versionControl/index.test.ts

```diff
@@ -1,25 +1,32 @@
+import { prompt } from 'inquirer';
+
 import { GeneralOptions } from '@/commands/generate';
 import { remove } from '@/helpers/file';
 
 import { applyVersionControl } from '.';
 
+jest.mock('inquirer');
+
 describe('Version control add-on', () => {
-  describe('given valid GeneralOptions', () => {
-    describe('given github version control', () => {
+  describe('given versionControlEnabled is true', () => {
+    describe('given github service', () => {
```

Comment:
Since Github is a name
```suggestion
    describe('given Github service', () => {
```